suite-native/app/.env.development

```diff
@@ -1,2 +1,3 @@
 EXPO_PUBLIC_ENVIRONMENT=debug
 EXPO_PUBLIC_IS_ANALYTICS_LOGGER_ENABLED=false
+EXPO_PUBLIC_IS_SENTRY_ENABLED=false
```

Comment:
I would like to have it on for development build it helps catch lot of stuff that happens during QA etc. It was like that since beginning.

Reply:
I see it is confusing, but it still works like that, because the condition for enabling is `enabled: !isDebugEnv() || process.env.EXPO_PUBLIC_IS_SENTRY_ENABLED === 'true',`. I'll add comment here.

Reply:
I'm changing the name to `EXPO_PUBLIC_IS_SENTRY_ON_DEBUG_BUILD_ENABLED` instead of the comment to make it more clear.

Reply:
f98d859b1134293234240ed9a66a50f53697fad6